assets/js/pages/SapSystemsOverviewPage/SapSystemsOverview.jsx

```diff
@@ -77,6 +77,22 @@ function SapSystemsOverview({
         title: 'Tenant',
         key: 'tenant',
       },
+      {
+        title: 'Type',
+        key: 'type',
```

Comment:
Here you cannot use `type` as we don't have this key in the data fields.
You need to use `applicationInstances`.
Once you do this, you will have `render: (content)` where content are the `applicationInstances`